verl/workers/fsdp_async_workers.py

```diff
@@ -0,0 +1,331 @@
+# Copyright 2024 Bytedance Ltd. and/or its affiliates
```

Comment:
Why this file is named with fsdp? Does it work with Megatron?

Reply:
For initial design, it's the counterpart of `fsdp_workers.py`, but the ExternalRayDistributedExecutor/AsyncLLMWorker/AsyncLLMManager should be also suitable to megatron. I will refactor it.

Reply:
Refactor `fsdp_async_workers.py` to `async_server.py`, it should be common to fsdp and megatron